HelloTorchIOS/HelloTorchIOS/LightsLogic.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  LightsLogic.swift
+//  HelloTorchIOS
+//
+//  Created by Luis Valle on 08/05/2018.
+//  Copyright © 2018 Novoda. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+class LightsLogic {
+    
+    private let device: AVCaptureDevice? = AVCaptureDevice.default(for: AVMediaType.video)
+    
+    init() {
+        do {
+            try device?.lockForConfiguration()
+        }catch {
+            print("Torch could not be used")
+        }
+    }
+    
+    func toggleStrobe() {
+        guard device?.hasTorch == true else {
```

Comment:
No need to check for `== true` it already defaults to checking for true.

Reply:
Not really, XCode complains because `hasTorch` seems to return an optional. If I remove the `== true` I get:
`Value of optional type 'Bool?' not unwrapped; did you mean to use '!' or '?'?`

Reply:
No yeah you are right, if it is an optional yeah then it makes sense.